README.md

```diff
@@ -75,41 +77,79 @@ STK can then be run from the build directory with `bin/supertuxkart`
 
 ### Further options
 
-To create a debug version of STK, use:
+To create a debug version of STK, run:
 
-```
+```bash
 cmake .. -DCMAKE_BUILD_TYPE=Debug
 ```
 
 You can install your build system-wide:
 
-```
+```bash
 sudo make install
 ```
 
-The default install location is `/usr/local`, i.e. the data files will
+By default, the install location is `/usr/local`, i.e. the data files will
 be written to `/usr/local/share/games/supertuxkart`, the executable
 will be copied to `/usr/local/bin`. To change the default installation
 location, specify `CMAKE_INSTALL_PREFIX` when running cmake, e.g.:
 `cmake .. -DCMAKE_INSTALL_PREFIX=/opt/stk`
 
 
-## Windows
 
-1. Install VS 2013 (or later). The free express versions work fine.
-2. Download and install a source package - either a released package or from our [git/svn repositories](https://supertuxkart.net/Source_control).
-3. Download the latest dependency package from [here](https://sourceforge.net/projects/supertuxkart/files/SuperTuxKart%20Dependencies/Windows/). Unzip it in the root directory, so that the dependencies directory is next to the src and data directories (if you are updating from a previous dependency package, you can delete the .dll files in the root directory, they are not needed anymore).
-4. Download cmake and install it. Then start cmake-gui and select the STK root directory as 'Where is the source code', and a new directory in the root directory (next to src, data etc) as the build directory (for now I assume that this directory is called bld).
-5. Click on configure. You will be asked to create the directory (yes), then for your VS version. Make sure you select the right version (be aware of the easy to confuse version numbers: VS 2013 = version 12). Click on configure, then generate. This will create the directory 'bld', and a VS solution in that directory.
-6. In Visual Studio open the project file generated in the 'bld' folder.
-7. Right click on the supertuxkart project in the solution explorer, and select "Set as StartUp Project".
-8. Select Build->Build Solution (or press F7) to compile.
+## Building SuperTuxKart from source on Windows
+To build SuperTuxKart on Windows, follow these instructions:
+
+0. Download and install Git for Windows from here: [Git for Windows - Download](https://git-scm.com/download/win), then download and install TortoiseSVN from here: [TortoiseSVN - Downloads](https://tortoisesvn.net/downloads.html)
+*Skip this step **if** Git for Windows and TortoiseSVN are already installed*
+1. Download and install Visual Studio from here: [Visual Studio - Download](https://www.visualstudio.com/downloads/). The free Visual Studio Community edition works fine.
+2. Download the SuperTuxKart source package from [SuperTuxKart.net - Source Control](https://supertuxkart.net/Source_control), and unpack it. 
```

Comment:
The files obtained source control are indeed not packed, that was a reference to the sf.net source packages that i forgot to remove when removing the references to the source packages

a commit that corrects this will be pushed out very soon